PolyEngine/Core/Src/Queue.hpp

```diff
@@ -79,7 +79,7 @@ namespace Poly
 		bool operator!=(const Queue<T>& rhs) const { return !(*this == rhs); };
 
 		//------------------------------------------------------------------------------
-		bool IsEmpty() const { return GetSize() == 0; }
+		bool Empty() const { return GetSize() == 0; }
```

Comment:
You've written "make it empty" in last change requests, and I misunderstood you. Thought you want me to rename this function, but you meant to make default ctor empty ;P